TA-microsoft-graph-security-score/bin/input_module_graph_security_score_input.py

```diff
@@ -14,7 +14,8 @@
 CLIENT_SECRET = "client_secret"
 LOCK_FILE = "not_first.lock"
 ADDON_NAME = "TA-microsoft-graph-security-score"
-LOCK_FILE_PATH = splunk_lib_util.make_splunkhome_path(["etc", "apps", ADDON_NAME, "bin", LOCK_FILE])
+LOCK_FILE_PATH = splunk_lib_util.make_splunkhome_path(["etc", "apps", ADDON_NAME, "local", "checkpoint", LOCK_FILE])
+LOCK_FILE_DIR = splunk_lib_util.make_splunkhome_path(["etc", "apps", ADDON_NAME, "local", "checkpoint"])
```

Comment:
```
LOCK_FILE_DIR = splunk_lib_util.make_splunkhome_path(["etc", "apps", ADDON_NAME, "local", "checkpoint"])
LOCK_FILE_PATH = os.path.join(LOCK_FILE_DIR, LOCK_FILE)
```

Reply:
@VatsalJagani It will ignore older checkpoint for first run but i think it's okay as the number of events are less

Reply:
yup, it's fine. I even had the impression, we don't have a checkpoint in this TA.